### ISSUE TITLE
Question regarding Equivariance loss

### ISSUE THREAD

**Issue by danushkv** [transmuteAI/implicit-networks#1]
Hey authors!!! Thank you for implementation of ```IEN``` in contract to steerable CNNs. My question is regarding the ```equivarince loss```. I understand that one term which goes into the loss, is the ```GroupPooled``` output from the layer. But, what exactly is the other term?
Let's say x1 is an image from the dataset X, then the GroupPooled output from the layer i is the first term. How exactly is the other term here defined? [lines](https://github.com/transmuteAI/implicit-networks/blob/6788084c2292a582b8204a951496c5f8901dd895/main.py#L149) found here
```python
temp_rot = torch.cat((temp_rot, augmented_temp.unsqueeze(0)),0)
loss2.append(equivariance_loss(temparr[j][:tbs],temp_rot))
```
